lib/solargraph/source_map.rb

```diff
@@ -159,6 +161,17 @@ def map source
 
     private
 
+    # @return [Array<Pin::Base>]
+    def convention_pins
+      @convention_pins || []
+    end
+
+    def convention_pins=(pins)
+      # unmemoizing the document_symbols in case it was called from any of convnetions
```

Comment:
For example, solargraph-rails:

https://github.com/iftheshoefritz/solargraph-rails/blob/144cec0d91028b8073029df0bbaf58f6a2adbb3b/lib/solargraph-rails.rb#L39

This leads to subsequent convention pins not being included, hence we need to clear the memorization after all convention pins are collected.